app.js

```diff
@@ -119,10 +120,22 @@
     }, container)
     container.className = "eager-hero-image"
 
-    container.addEventListener("click", () => scrollToTop({
-      element: parent,
-      finalY: container.clientHeight
-    }))
+    if (options.redirect) {
```

Comment:
The logic inside the `if` statement is accurate but can be simplified.

Create one event listener and have the `if` statement inside.